src/Groups/homomorphisms.jl

```diff
@@ -622,18 +681,32 @@ function isomorphism(::Type{T}, A::FinGenAbGroup) where T <: GAPGroup
        A2, A2_to_A = snf(A)
      end
      A_to_A2 = inv(A2_to_A)
-     # the isomorphic gap group
-     G = abelian_group(T, exponents)
-     # `GAPWrap.GeneratorsOfGroup(G.X)` consists of independent elements
+     # Create an isomorphic GAP group whose `GAPWrap.GeneratorsOfGroup`
+     # consists of independent elements of the orders in `exponents`.
+     # (We cannot guarantee that these generators form a pcgs in the case
+     # `T == PcGroup`, hence we cannot call `abelian_group(T, exponents)`.)
+     if T == PcGroup
+       if 0 in exponents
+         GapG = GAP.Globals.AbelianPcpGroup(length(exponents), GAP.GapObj(exponents, recursive=true))
```

Comment:
I guess we can drop the `GAP.` from `GAP.GapObj` here and elsewhere? Then again we still use it in a bunch of places, so we could change that in a later follow-up PR.
```suggestion
         GapG = GAP.Globals.AbelianPcpGroup(length(exponents), GapObj(exponents, recursive=true))
```

Reply:
Yes, we can write just `GapObj` because meanwhile it is an exported variable.